test/controllers/static_pages_controller_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'test_helper'
+
+class StaticPagesControllerTest < ActionDispatch::IntegrationTest
+  
+  # Test setup
+  def setup
+    @base_title = 'Ruby on Rails Tutorial Sample App'
+  end
+  
+  # Test index
+  #test "should get root" do
+  #  get FILL_IN
+  #  assert_response FILL_IN
+  #end
+  
+  
+  test 'should succeed get home' do
+    get static_pages_home_path
+    assert_response :success
+    assert_select 'title', "Home | #{@base_title}"
+  end
+
+  test 'should succeed get help' do
+    get static_pages_help_path
+    assert_response :success
+    assert_select 'title', "Help | #{@base_title}"
+  end
+  
+  test 'should succeed get about' do
+    get static_pages_about_path
+    assert_response :success
+    assert_select 'title', "About | #{@base_title}"
```

Comment:
Above everything, be consistent. So if you're going to `assert_select` the title here, do it in the Micropost controller tests for index, show, edit